packages/create-atomic/plopfile.ts

```diff
@@ -0,0 +1,62 @@
+import {NodePlopAPI} from 'plop';
+import {spawn} from 'child_process';
+import {getPackageManager} from './utils';
+
+interface PromptsAnswers {
+  project: string;
+}
+
+module.exports = function (plop: NodePlopAPI) {
+  const currentPath = process.cwd();
+  plop.setGenerator('@coveo/atomic', {
+    description: 'A Coveo Atomic Generator',
+    prompts: [
+      {
+        type: 'input',
+        name: 'project',
+        message: 'Name of the project',
+        validate: (input) => {
+          // Taken from http://json.schemastore.org/package
+          const npmPackageRegex =
+            '^(?:@[a-z0-9-*~][a-z0-9-*._~]*/)?[a-z0-9-~][a-z0-9-._~]*';
+          if (!new RegExp(npmPackageRegex).test(input)) {
+            return 'Project name is invalid, please follow the guidelines: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#name)';
+          }
+
+          return true;
+        },
+      },
+    ],
+    actions: [
+      {
+        type: 'add',
+        path: currentPath + '/{{project}}/package.json',
+        templateFile: 'templates/package.hbs',
+      },
+      {
+        type: 'add',
+        path: currentPath + '/{{project}}/public/index.html',
+        templateFile: 'templates/public/index.hbs',
+      },
+      {
+        type: 'addMany',
+        destination: currentPath + '/{{project}}/',
+        base: 'templates',
+        templateFiles: [
+          'templates/public/style/*',
+          'templates/scripts/*',
+          'templates/.env.example',
+          'templates/.gitignore',
+        ],
+      },
+      function installPackages(answers) {
+        const {project} = answers as PromptsAnswers;
```

Comment:
Wouldn't that work too?
```suggestion
      function installPackages(answers: PromptAnswers) {
        const {project} = answers;
```

Reply:
<img width="830" alt="Screen Shot 2021-11-11 at 12 04 22 PM" src="https://user-images.githubusercontent.com/4923043/141338973-f46de2a8-a0da-44a1-b16c-e11ef52197ae.png">
 Nope, plop's type are 💩

Reply:
:feelsgood: oh well ✅